salt/buildbot/init.sls

```diff
@@ -49,6 +49,16 @@ buildbot-user:
       - user: buildbot-user
       - file: /srv
 
+{# vagrant up fails due to this file not existing (see: https://github.com/python/psf-salt/pull/367#issuecomment-2218636563) #}
```

Comment:
im not sure this is the right move, but it fixed the provisioner

Reply:
Notably, when I brought this up on the default, `focal`, it gave the same error... So I'm more inclined to just leave this. I've not yet dug into what eventually touches this in https://github.com/python/buildmaster-config, but this shouldnt hurt much?